src/client/mapblock_mesh.cpp

```diff
@@ -732,6 +732,21 @@ MapBlockMesh::MapBlockMesh(Client *client, MeshMakeData *data):
 	m_has_animation =
 		!m_crack_materials.empty() ||
 		!m_animation_info.empty();
+
+	m_any_edge_nodes = false;
+	const auto bpn = data->m_blockpos * MAP_BLOCKSIZE;
+	const auto sl_1 = data->m_side_length - 1;
+	for (u16 z = 0; z < data->m_side_length; ++z)
+	for (u16 y = 0; y < data->m_side_length; ++y)
+	for (u16 x = 0; x < data->m_side_length; ++x) {
+		if ((x != 0 && x != sl_1) || (y != 0 && y != sl_1) || (z != 0 && z != sl_1))
```

Comment:
This is only false at the 8 corners of the cube, no? Probably not what you want. :)

Reply:
Hmm right. What I want to do is check all outer nodes, but the current condition is indeed broken.
I'll convert this PR to a simpler variant so we can merge this as a simple improvement for 5.13.